test/test_rosbag/test/record_cleanup.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
```

Comment:
Please license the new file under BSD since that is the license of the package and it doesn't make much sense to have this single file have a different license.

Reply:
Will fix.